.gitignore

```diff
@@ -24,9 +23,11 @@ docs/
 
 # Development tool files
 *.swp
-/local/
```

Comment:
Please don't remove `/local/` in `.gitignore`. This is a good place for anyone of us to store arbitrary local files that should not be committed.

Reply:
Ups, I deleted it by mistake, thanks